wazuh/wazuh-kibana-app
# Fix the agent graph in opensearch dashboard

### Description

### This PR only applies to 4.4-2.3-wzd, no backports required.

Fixed agent graph in opensearch dashboard.
 
### Issues Resolved

#4928

### Evidence

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/99441266/205046548-a77741e0-be8e-46cf-a878-5fcd5f83c948.png">

### Steps to test

*Click on agents 
*Verify that the graph that shows the agents status is correctly displayed.

### Check List
